File: footie/FootieBlitz/src/model/ScoutAssignment.java
package model;

public class ScoutAssignment {

	public int id, userId, personId, countryId, hoursSinceStart, detailLevel;
	
	public ScoutAssignment(int id, int userId, int personId, int countryId, int hoursSinceStart, int detailLevel) {
		super();
		this.id = id;
		this.userId = userId;
		this.personId = personId;
		this.countryId = countryId;
		this.hoursSinceStart = hoursSinceStart;
		this.detailLevel = detailLevel;
	}
	
}

File: footie/FootieBlitz/website/test/Model/ScoutAssignment.php
<?php

class ScoutAssignment extends AppModel
{
 var $name = 'ScoutAssignment';
 
    var $belongsTo = array(
        'User' => array(
			'className' => 'User'
		),
		'Person' => array(
			'className' => 'Person'
		),
		'Country' => array(
			'className' => 'Country'
		)
    );
    
    var $hasMany = array(
            'ScoutReport' => array(
    			'className' => 'ScoutReport',
    			'order' => 'potential DESC'
    		)
    );
}

//CREATE TABLE scout_assignments (
//id serial primary key, 
//club_id int references clubs(id), 
//user_id int references users(id), 
//country_id int references countries(id), 
//person_id int references persons(id), 
//start timestamp default now(), 
//finished timestamp);
?>